examples/seismic/model.py

```diff
@@ -499,9 +499,10 @@ def __init__(self, origin, spacing, shape, space_order, nbl=20,
         self.grid = Grid(extent=extent, shape=shape_pml, origin=origin_pml, dtype=dtype,
                          subdomains=subdomains)
 
-        # Create dampening field as symbol `damp`
-        self.damp = Function(name="damp", grid=self.grid)
-        initialize_damp(self.damp, self.nbl, self.spacing, mask=damp_mask)
+        if self.nbl != 0:
+            # Create dampening field as symbol `damp`
+            self.damp = Function(name="damp", grid=self.grid)
+            initialize_damp(self.damp, self.nbl, self.spacing, mask=damp_mask)
```

Comment:
still needs `else` part
```
else:
    self.damp = 1 if damp_mask else 0
```
so that examples work too with nbl=0 e.g:
`python examples/seismic/acoustic/acoustic_example.py -nbl 0`

Reply:
Ah yes, thanks.